marketplace-ui/src/app/shared/constants/common.constant.ts

```diff
@@ -36,21 +36,25 @@ export const NAV_ITEMS: NavItem[] = [
 
 export const SOCIAL_MEDIA_LINK = [
   {
-    styleClass: 'fab fa-linkedin',
-    url: '/'
+    styleClass: 'fa-brands fa-linkedin-in',
```

Comment:
add class `text-primary` to change the text color to `#1B1B1B`
![image](https://github.com/user-attachments/assets/0f20e348-5265-454f-a0e1-e4fcd1a58f71)
-> `fa-brands fa-linkedin-in text-primary`
Adapt for other link icons

Reply:
Added text-primary for correct footer icon color